src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/AnnotationUtilsUnitTest.java

```diff
@@ -0,0 +1,170 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator;
+
+import htsjdk.samtools.Cigar;
+import htsjdk.samtools.CigarElement;
+import htsjdk.samtools.CigarOperator;
+import org.broadinstitute.hellbender.utils.read.ArtificialReadUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+import static org.testng.Assert.*;
+
+public class AnnotationUtilsUnitTest {
```

Comment:
I can check these later if you'd like, but I figured I'd get comments on the rest back to you faster if I don't do it now.